DATA_SOURCES.md

```diff
@@ -389,6 +389,7 @@ Cross-border transmission capacities between the zones are centralized in the [e
 - Åland ⇄ Sweden: ["Sverigekabeln": 80 MW](https://kraftnat.ax/stamnatet/)
 - Åland ⇄ Finland: ["Gustavskablen": 10 MW + "ÅL-Link": 100 MW](https://kraftnat.ax/stamnatet/)
 - Albania ⇄ Greece: [533 MW](https://ec.europa.eu/energy/sites/ener/files/documents/2nd_report_ic_with_neighbouring_countries_b5.pdf)
+- Andorra ⇄ Spain: [125 MW](https://www.ree.es/sites/default/files/publication/2022/05/downloadable/inf_sis_elec_ree_2020_0EN_0.pdf)
```

Comment:
Looks good to me but you should add the actual exchange (with capacity) to `exchanges.json` as well.